theta/include/theta_sketch.hpp

```diff
@@ -106,6 +107,7 @@ class base_theta_sketch_alloc {
   virtual void print_items(std::ostringstream& os) const = 0;
 };
 
+/// Base class for the Theta Sketch, a generalization of the Kth Minimum Value (KMV) sketch.
```

Comment:
Is it a generalization of or a type of KMV sketch? Maybe sidestep and describe it as a type of KMV sketch for counting distinct items?

Reply:
we have this on the web site: "Theta Sketches are a generalization of the well known Kth Minimum Value (KMV)1,2 sketches in that KMV sketches are a form of Theta Sketch, but not all Theta Sketches are KMV."

Reply:
Ok, although a bit misleading in the context of C++ where we don't offer anything but KMV. That's fine though.